tests/Object/Users/MergeIdentifierTest.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+namespace ImmobiliareLabs\BrazeSDK\Test\Object\Users;
+
+use ImmobiliareLabs\BrazeSDK\Exception\ValidationException;
+use ImmobiliareLabs\BrazeSDK\Object\UserAlias;
+use ImmobiliareLabs\BrazeSDK\Object\Users\MergeIdentifier;
+use ImmobiliareLabs\BrazeSDK\Object\Users\NewUserAlias;
+use PHPUnit\Framework\TestCase;
+
+class MergeIdentifierTest extends TestCase
+{
+    /**
+     * @dataProvider validProvider
+     * @doesNotPerformAssertions
+     */
+    public function testValid(MergeIdentifier $mergeIdentifier): void
+    {
+        $mergeIdentifier->validate(true);
```

Comment:
How does this test work? I don't see any asserts.

Reply:
The `validate` method throws an exception if a validation error occurs. So, if test pass, it means that validation correctly does not throws exceptions with the provided valid object.

Reply:
No, we can't. The `validate` method is called on a valid object, so no exceptions are thrown.